### PR TITLE
New page: aria-details

### PR DESCRIPTION
**Pull request by estelle**
- [X] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by ericwbailey — CHANGES_REQUESTED** [review]
There's some pretty large editorial changes that I'm proposing here. Happy to chat about it if you'd like, @estelle.

File: files/en-us/web/accessibility/aria/attributes/aria-details/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+```html
+<p>The <strong>cubic-bezier()<strong> functional notation defines a cubic 
+  <span role="term" aria-details="bezier bezImg">Bézier curve</span>. As 
+  these curves are continuous, they are often used to smooth down the start and 
+  end of the curve and are therefore sometimes called easing functions.
+</p>
+
+<p role="definition" id="bezier">A <strong>Bézier curve</strong>,
+ (pronounced /<span title="primary stress syllable">ˈb<span title="/ɛ/: 'e' in 'dress'">ɛ</span>z'</span>-y-<span title="/eɪ/: 'a' in 'face'">eɪ</span>/ 
+ <i title="English pronunciation respelling">BEH-zee-ay</i>) is a mathematically
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-details/index.md:43]: i wanted to show "semantics" forcing the use of `aria-details` over `aria-describedby`.

File: files/en-us/web/accessibility/aria/attributes/aria-details/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ <i title="English pronunciation respelling">BEH-zee-ay</i>) is a mathematically 
+ described curve used in computer graphics and animation. The curve is defined 
+ by a set of control points with a minimum of two. Web related graphics 
+ and animations use Cubic Béziers, which are curves with four control 
+ points P<sub>0</sub>, P<sub>1</sub>, P<sub>2</sub>, and P<sub>3</sub>.
+</p>
+
+<a href="bezierExplanation.html" id="bezImg" 
+  aria-label="Explanation of Bézier curve in CSS timing functions">
+  <img alt="Animated Bézier curve" src="bezier.gif">
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-details/index.md:52]: the goal was to show a link to a different page as part of details.

File: files/en-us/web/accessibility/aria/attributes/aria-details/index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+The elements referenced by `aria-details` should be visible to all users. `aria-details` informs users that otherwise might not be able to scan a screen and discern quickly that the explanatory content is available.
+
+> **Note:**  `aria-details` has no impact on the accessible description. 
+
+Unlike `aria-describedby`, elements referenced by `aria-details` are not used in accessible descriptions and are not stringified when presented to assistive technology users. If the associated content is not too long and flattening the contents of the referenced element to a simple string of text wouldn't cause loss of information, consider using `aria-describedby` instead. That said, it is valid for an element to have both `aria-details` and a description specified with either `aria-describedby` or `aria-description`.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-details/index.md]: updated in a separate commit

File: files/en-us/web/accessibility/aria/attributes/aria-details/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ericwbailey) [reply files/en-us/web/accessibility/aria/attributes/aria-details/index.md:56]: Yes, but I think we should also be striving to have meaningful alt descriptions in code examples to model for anyone reading.

File: files/en-us/web/accessibility/aria/attributes/aria-details/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ericwbailey) [reply files/en-us/web/accessibility/aria/attributes/aria-details/index.md:47]: Yup, got that. This use of the `title` attribute is an antipattern, however.

[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-details/index.md]: i added an aria-description.

File: files/en-us/web/accessibility/aria/attributes/aria-details/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-details/index.md]: made a more detailed alt attr.

File: files/en-us/web/accessibility/aria/attributes/aria-details/index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+```html
+<p>The <strong>cubic-bezier()<strong> functional notation defines a cubic 
+  <span role="term" aria-details="bezier bezImg">Bézier curve</span>. As 
+  these curves are continuous, they are often used to smooth down the start and 
+  end of the curve and are therefore sometimes called easing functions.
+</p>
+
+<p role="definition" id="bezier">A <strong>Bézier curve</strong>,
+ (Prounounced \ ˈbe-zē-ˌā \)
+ <i title="English pronunciation respelling" aria-description="English pronunciation">BEH-zee-ay</i>) is a mathematically
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/aria/attributes/aria-details/index.md:47]:
```suggestion
 <i aria-description="English pronunciation">BEH-zee-ay</i>) is a mathematically 
```

File: files/en-us/web/accessibility/aria/attributes/aria-details/index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+ <i title="English pronunciation respelling" aria-description="English pronunciation">BEH-zee-ay</i>) is a mathematically 
+ described curve used in computer graphics and animation. The curve is defined 
+ by a set of control points with a minimum of two. Web related graphics 
+ and animations use Cubic Béziers, which are curves with four control 
+ points P<sub>0</sub>, P<sub>1</sub>, P<sub>2</sub>, and P<sub>3</sub>.
+</p>
+
+<a href="bezierExplanation.html" id="bezImg" 
+  aria-label="Explanation of Bézier curve in CSS timing functions">
+  <img alt="Animated Bézier curve showing 4 control points" src="bezier.gif">
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/aria/attributes/aria-details/index.md:56]:
```suggestion
  <img alt="Animated Bézier curve showing 4 control points." src="bezier.gif">
```

**Review by ericwbailey — CHANGES_REQUESTED** [review]

**Review by ericwbailey — APPROVED** [review]

**Pull request merged**